test/lens_odm_parser/core_test.clj

```diff
@@ -110,7 +119,7 @@
       Double/NaN)))
 
 (defmacro given-parsed [sexp & body]
-  `(given (parse (element ~sexp))
+  `(given (-> ~sexp ~'element ~'parse-elem ~'first ~'second ~'first)
```

Comment:
If you output symbols without namespace here, given-parsed works only because they are in context while the resulting macro code is executed. It's better to directly refer to the symbols in context at macro expansion time, so that the symbols are fully resolved in the resulting code.